eng/pipelines/runtime.yml

```diff
@@ -448,8 +448,8 @@ extends:
           - linux_x86
           jobParameters:
             testScope: innerloop
-            nameSuffix: CoreCLR_NoR2R
-            buildArgs: -s clr.runtime+clr.jit+clr.iltools+clr.spmi+clr.corelib -c $(_BuildConfig)
+            nameSuffix: CoreCLR
+            buildArgs: -s clr.runtime+clr.jit+clr.iltools+clr.spmi+clr.corelib+clr.nativecorelib -c $(_BuildConfig)
```

Comment:
There is a comment above that looks like it should be updated if we are starting to run linux-x86 crossgen2 with this PR